### DISCUSSION TITLE
running Out-HTMLView without it opening a browser

### DISCUSSION THREAD

**Discussion by kevinelwell** [EvotecIT/PSWriteHTML#414] [Q&A] [answered]
Fantastic module! I am using it to create a report of all the VMs in my ESXi clusters. I would like the end result to be onyl the .html report is created and does not open a browser at the end. I am using PSWriteHTML module version 0.0.174.

Also is there a way to have more than 22 results per page in the .html  output without needing to click on the next page? Can I define how many results I want per page?

Thank you!

Kevin

**Comment by PrzemyslawKlys** [comment, marked as answer]
There's plenty of options:

![image](https://github.com/EvotecIT/PSWriteHTML/assets/15063294/7defb218-6a0e-4c9c-bd9c-4b3afe080ed9)

```powershell
Get-Process | Out-HtmlView -PreventShowHTML -PagingOptions 10,50,100 -PagingLength 50
```

I would advice you to read blog posts on: https://evotec.xyz/?s=pswritehtml to understand more on what options are there and review examples on GitHub, as there are plenty. Out-HtmlView while having 40 parameters is just a tip what it can do. Of course with PreventHTML you need to provice FilePath
